src/components/matrix/grid-cell.tsx

```diff
@@ -0,0 +1,89 @@
+import { memo, CSSProperties } from 'react';
+import { areEqual } from 'react-window';
+
+export type GridCellProps = {
+  columnIndex: number;
```

Comment:
As I said I'm not going to comment on each individual component and I expect you to comment what each prop is. Even if it looks obvious, please add it for completeness. We might also eventually add more less obvious props, and that would allow us to consistently comment everything.

Reply:
added for most exported types for now